src/translations/src/cboard.json

```diff
@@ -91,6 +91,15 @@
   "cboard.components.Board.emptyVoiceAlert": "WARNING: we did not detect an available Text to Speech voice! Cboard cannot work properly.",
   "cboard.components.Board.myBoardTitle": "My Board",
   "cboard.components.Board.failedToCopy": "Failed to copy to clipboard",
+  "cboard.components.Board.walkthroughWelcome": "Welcome to Cboard!",
+  "cboard.components.Board.walkthroughUnlock": "Press the lock 4 times to unlock your options and settings!",
```

Comment:
"Press the lock 4 times to unlock your options and settings!"
Should be:

"Press the lock button four times to unlock your options and settings!"

Reply:
I modified the messages and replaced all "!" with ".", except from the welcome message ("Welcome to Cboard!")